edge-functions/feature-flag-posthog/lib/posthog-node.ts

```diff
@@ -33,10 +33,14 @@ export async function isFeatureFlagEnabled(
  *          If the feature flag is a multvariate, then the value will be a string
  */
 export async function getFeatureFlagVariant(
-  distinctUserId: string,
+  distinctUserId: string = '',
```

Comment:
Likely unwanted to make this field optional based on the check below about it not being empty